### PR TITLE
Added apoc.text.charAt() and apoc.text.hexCharAt()

### PR DESCRIPTION
**Pull request by InverseFalcon**
Fulfills request #723

### CODE REVIEW

File: src/main/java/apoc/text/Strings.java [hunk 1]
```diff
@@ -372,4 +372,14 @@ public String base64Decode(@Name("text") String text) {
         byte[] decoded = Base64.getDecoder().decode(text.getBytes());
         return new String(decoded);
     }
+
+    @UserFunction
+    @Description("apoc.text.charAt(text, index) - the hex value string of the character at the given index")
+    public String charAt(@Name("text") String text, @Name("index") Long index) {
```
Comment (jexp) [inline src/main/java/apoc/text/Strings.java]: shouldn't this return the long/int value?
and then a separate function do the toHex ?

Comment (InverseFalcon) [reply src/main/java/apoc/text/Strings.java]: Could do that. Ran into some issues with `Character.getNumericValue()` not returning the correct values for certain characters, I'll check for alternatives.

**Comment by InverseFalcon** [comment]
For hexCharAt, does the empty string return make sense for invalid inputs, or would null work better?

**Comment by jexp** [comment]
Merged manually and added apoc.text.hexValue

**Pull request closed without merging**